torchtune/models/flamingo/_transform.py

```diff
@@ -100,6 +100,7 @@ def __init__(
         self.stop_tokens = self.tokenizer.stop_tokens
         self.max_seq_len = max_seq_len
         self.prompt_template = prompt_template
+        self.pad_id = self.tokenizer.pad_id
```

Comment:
I know there's not a clear way around it, but I don't love that we have to do this. It'll be a gotcha for anyone trying to write their own transforms

Reply:
This is a way around this, but it involves inheriting from the tokenizer